build_tools/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Add the current dir to PYTHONPATH to simulate running environment of tools.
+set(ENV{PYTHONPATH} ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
What is this for? Typically we don't want the build system to secretly alter important environment variables like `$PATH` and `$PYTHONPATH`. It may cause subtle issues. Also this is dropping all existing stuff from `$PYTHONPATH`?

Reply:
This is a temporary solution before we have a proper python test rule for non python binding code. Without it, the tests can't find the `common` package under `build_tools/benchmarks`.

I filed an issue #8708 to track it and updated the comment to explain it.

Included the existing PYTHONPATH to the new path, thanks for catching that!

Reply:
All our current python tests alter `PYTHONPATH` ftr

Reply:
Ah... but this is setting PYTHONPATH for the whole cmake process and children, not just for an individual test. That is more concerning. Could you instead set the env properties of the tests? You can do that out of band after using the `iree_py_test` function. It's a hack, but a more localized one than this :-)

Reply:
Added python path for each test instead of the whole environment.

I'm not super familiar with python package system and cmake rules, so let me know if there is a better way : )